tests/expectations/core/test_core_model_schemas.py

```diff
@@ -22,6 +22,7 @@ def test_all_core_model_schemas_are_serializable():
         model.schema_json()
 
 
+@pytest.mark.slow  # 4.2s
```

Comment:
`unit` will force this to have an upper bound of 2s. If this test takes longer you should use a different marker from [this list](https://github.com/great-expectations/great_expectations/blob/develop/tests/conftest.py#L112). Every test must have exactly 1 of these markers. If there is a reason it's slow, like it touches the filesystem (I think this test does?) use `pytest.mark.filesystem`. If there is no reason lining up with a particular marker you can use `big`.

Reply:
I don't think if the `slow` marker actually does anything anymore?

Reply:
Ah filesystem worked, ty